spinnaker_camera_driver/rviz/cam_test.rviz

```diff
@@ -24,132 +24,41 @@ Panels:
   - Class: rviz/Time
     Name: Time
     SyncMode: 0
-    SyncSource: /stereo/color/image_color
+    SyncSource: /usb_camera/image_color
 Preferences:
   PromptSaveOnExit: true
 Toolbars:
   toolButtonStyle: 2
 Visualization Manager:
   Class: ""
   Displays:
-    - Class: rviz/TF
-      Enabled: true
-      Frame Timeout: 15
-      Frames:
-        All Enabled: true
-      Marker Alpha: 1
-      Marker Scale: 1
-      Name: TF
-      Show Arrows: true
-      Show Axes: true
-      Show Names: true
-      Tree:
-        {}
-      Update Interval: 0
-      Value: true
-    - Class: rviz/Image
-      Enabled: true
-      Image Topic: /stereo/left/image_raw
-      Max Value: 1
-      Median window: 5
-      Min Value: 0
-      Name: /stereo/camera_left
-      Normalize Range: true
-      Queue Size: 2
-      Transport Hint: raw
-      Unreliable: false
-      Value: true
-    - Class: rviz/Image
-      Enabled: true
-      Image Topic: /stereo/right/image_raw
-      Max Value: 1
-      Median window: 5
-      Min Value: 0
-      Name: /stereo/camera_right
-      Normalize Range: true
-      Queue Size: 2
-      Transport Hint: raw
-      Unreliable: false
-      Value: true
-    - Class: rviz/Image
-      Enabled: true
-      Image Topic: /stereo/left/image_rect
-      Max Value: 1
-      Median window: 5
-      Min Value: 0
-      Name: /stereo/left/image_rect
-      Normalize Range: true
-      Queue Size: 2
-      Transport Hint: raw
-      Unreliable: false
-      Value: true
     - Class: rviz/Image
       Enabled: true
-      Image Topic: /stereo/right/image_rect
+      Image Topic: /stereo/lower/image_raw
       Max Value: 1
       Median window: 5
       Min Value: 0
-      Name: /stereo/right/image_rect
+      Name: /stereo/lower/image_raw
       Normalize Range: true
       Queue Size: 2
       Transport Hint: raw
       Unreliable: false
       Value: true
-    - Alpha: 1
-      Autocompute Intensity Bounds: true
-      Autocompute Value Bounds:
-        Max Value: 19.730342864990234
-        Min Value: 5.59285306930542
-        Value: true
-      Axis: Z
-      Channel Name: intensity
-      Class: rviz/PointCloud2
-      Color: 255; 255; 255
-      Color Transformer: MONO8
-      Decay Time: 0
-      Enabled: false
-      Invert Rainbow: false
-      Max Color: 255; 255; 255
-      Min Color: 0; 0; 0
-      Name: /stereo/disparity/points
-      Position Transformer: XYZ
-      Queue Size: 10
-      Selectable: true
-      Size (Pixels): 2
-      Size (m): 0.10000000149011612
-      Style: Points
-      Topic: /stereo/points2
-      Unreliable: false
-      Use Fixed Frame: true
-      Use rainbow: true
-      Value: false
     - Class: rviz/Image
       Enabled: true
-      Image Topic: /stereo/color/image_color
+      Image Topic: /stereo/upper/image_raw
```

Comment:
I deleted those topics from the `.rviz` config because it isn't needed here. The images from the primary / secondary topics are displayed in the `manager` rviz instance. 

I just view this `cam_test.rviz` as something that should be called for dumb tests